components/Card.vue

```diff
@@ -1,13 +1,34 @@
 <template>
-  <v-card :to="`../item/${item.Id}`">
-    <v-img class="cardImage" :src="imageLink(item.Id)" />
-    <v-card-title>
-      <span>{{ item.Name }}</span>
-    </v-card-title>
-    <v-card-subtitle>
-      <span>{{ item.ProductionYear }}</span>
-    </v-card-subtitle>
-  </v-card>
+  <nuxt-link
+    :to="`/item/${item.Id}`"
+    style="text-decoration: none; color: inherit"
+  >
+    <div class="card-box">
+      <div :class="cardType">
+        <button
+          class="card-content card-content-button d-flex justify-center align-center"
+          :style="{ backgroundImage: `url('${imageLink(item.Id)}')` }"
+        >
+          <v-icon
+            v-if="!item.ImageTags.Primary"
+            size="96"
+            color="grey lighten-3"
+          >
+            {{ itemIcon }}
+          </v-icon>
+        </button>
+        <div class="card-overlay d-flex justify-center align-center">
+          <v-btn fab color="primary" :to="`/item/${item.Id}/play`">
+            <v-icon size="36">mdi-play</v-icon>
+          </v-btn>
+        </div>
```

Comment:
We probably want some sort of v-if to prevent this from being visible on non-touch devices

Reply:
Added a media query for touch devices. We'll have to figure out how to handle it once we start making this work on TVs, but it's good enough for now :)

Reply:
Sounds good for now